qa/tasks/cephfs/test_scrub_checks.py

```diff
@@ -56,19 +81,20 @@ def test_scrub_abort(self):
         self.assertTrue("no active" in out_json['status'])
 
         # sleep enough to fetch updated task status
-        time.sleep(10)
-        self._check_task_status("idle")
+        checked = self._check_task_status_na()
+        self.assertTrue(checked)
 
     def test_scrub_pause_and_resume(self):
         test_dir = "scrub_control_test_path"
         abs_test_path = "/{0}".format(test_dir)
 
+        success_validator = lambda j, r: TestScrubChecks.json_validator(j, r, "return_code", 0)
```

Comment:
`./tasks/cephfs/test_scrub_checks.py:91:9: F841 local variable 'success_validator' is assigned to but never used`

Reply:
ack